### ISSUE TITLE
New twitter messes with the GH logo image display in the twitter card

### ISSUE THREAD

**Issue by stefaniebutland** [ropensci/roweb2#499]
Tweet from Bob Rudis https://twitter.com/hrbrmstr/status/1155104691182694401

Implementation of twitter cards closed issue https://github.com/ropensci/roweb2/issues/290

**Comment by jeroen** [comment]
What do other organizations have? Usually you want transparency specifically because it looks nice on different backgrounds.

**Comment by maelle** [comment]
Right. The tidyverse and r-lib ones have transparent background actually, and they're so dark they probably look as bad as the ropenscilabs one in Twitter night mode. :woman_shrugging:
